operator/config/crd/patches/webhook_in_lokistacks.yaml

```diff
@@ -0,0 +1,18 @@
+# The following patch enables a conversion webhook for the CRD
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  name: lokistacks.loki.grafana.com
+spec:
+  conversion:
+    strategy: Webhook
+    webhook:
+      clientConfig:
+        service:
+          namespace: system
+          name: webhook-service
```

Comment:
Should this be `loki-operator-webhook-service` instead?

Reply:
Yeah this was a mistake by the broken `config/crd/kustomizeconfig.yaml` that referenced only CRDs in v1beta1.